src/helpers/renderUnitHelper.js

```diff
@@ -39,33 +39,33 @@ export const renderUnitFilmDate = (unit, t) => {
   return filmDate ? renderFilmDate(filmDate) : null;
 };
 
-export const renderUnitNameAsListItem = unit => 
-  <List.Item key={unit.id} as={Link} to={canonicalLink(unit)}>
-    {unit.name || NO_NAME}
+export const renderUnitNameAsListItem = unit =>
+  <List.Item key={unit?.id} as={Link} to={canonicalLink(unit)}>
```

Comment:
That's not correct to have key with value `undefined`. Better return `null` if unit is undefined

Reply:
ok